### DISCUSSION TITLE
[Remix] Parameterized Route configuration in remix.config.js working locally but not working in Vercel

### DISCUSSION THREAD

**Discussion by EdgarMachado** [vercel/community#5189] [Help]
### Summary

Hi!
I followed the documentation (https://remix.run/docs/en/main/file-conventions/remix-config#routes) to configure a route schema and it worked as expected locally. But in Vercel, the routes with parameters are returning 404 Not Found.

any suggestions?

### Steps to Reproduce

Here is a fragment of the routes function:

```javascript

  routes(defineRoutes) {
    return defineRoutes((route) => {
      route("/", "routes/index.tsx", { index: true });    //OK!

      route("admin/", "routes/admin/index.tsx", () => {        //OK!

        route("home/", "routes/home/_index.tsx", () => {       //OK!

          route("", "routes/home/index.tsx", { index: true });     //OK!
        });

        route("person/", "routes/person/_index.tsx", () => {       //OK!

          route("", "routes/person/index.tsx", { index: true });     //OK!

          route("edit/:id", "routes/person/$id.tsx");   //NOT WORKING! 404 Not Found

          route("editV2/:id", "routes/person/edit.tsx");   //NOT WORKING! 404 Not Found
        });
      });
    });
  },

``` 

The error from log:

sn {
  status: 404,
  statusText: 'Not Found',
  internal: true,
  data: 'Error: No route matches URL "/admin/person/edit/1"',
  error: Error: No route matches URL "/admin/person/edit/1"
    at (node_modules/@remix-run/router/dist/router.js:3940:0)
    at (node_modules/@remix-run/router/dist/router.js:2849:0)
    at (node_modules/@remix-run/server-runtime/dist/esm/server.js:164:0)
    at (node_modules/@remix-run/server-runtime/dist/esm/server.js:89:0)
}

**Comment by NicoCPH** [comment]
Hello, i cant even deploy a project with a simple route config like so: 

```
  routes(defineRoutes) {
    return defineRoutes((route) => {
    route("/", "routes/_index.tsx", { index: true });
    route("about", "routes/about.tsx");
    });
    },
```
    

  How does your remix.config. js look like?
